JMS/README.md

```diff
@@ -225,6 +225,16 @@ If you have used maven to build the samples, you can run
 
 The response sample will get a message from the queue, process it and put the response on the reply to queue and keep looking for more messages to respond to till you ctrl+c interrupt it.
 
+### Poison Messages
+While running the request/response samples, if an error, such as "Reply to Queue no longer exists, skipping request" or "Reply to destination is invalid", is received then the message might be a Poison Message. This occurs when the message cannot get processed by the receiving application and gets sent back to the queue. The message will sit in the queue and the application will keep on seeing it until it's removed. This will become an endless loop.
```

Comment:
The error in this case is application specific. While this doc works for this application I think we should generalise in the explanation for a poison message scenario e.g., if the message will never be processed for some reason. In this case it is MQRC_UNKNOWN_OBJECT_NAME. The doc should reference the root cause.